src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const createDomNode = (type) => document.createElement(type);
+
+const addClassesToDomNode = (node, classes) => node.classList.add(...classes);
+
+const addTextToDomNode = (node, text) => (node.textContent = text);
+
+const appendNode = (parentNode, node) => parentNode.append(node);
+
+const createNotification = (type, parentNode, classes, text) => {
+  const node = createDomNode(type);
+
+  addClassesToDomNode(node, classes);
+
+  addTextToDomNode(node, text);
+
+  appendNode(parentNode, node);
+};
+
+const promise1 = new Promise((resolve) => resolve('Promise was resolved!'));
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error('Promise was rejected!')), 3000);
+});
+
+logo.addEventListener('click', () => {
+  promise1
+    .then((res) => {
+      createNotification('div', body, ['message'], res);
+    })
+    .catch((_error) => {
+      createNotification(
+        'div',
+        body,
+        ['message', 'error-message'],
+        _error.message,
+      );
+    });
+});
+
+promise2.catch((_error) => {
+  createNotification('div', body, ['message', 'error-message'], _error.message);
+});
```

Comment:
The `catch` block for `promise2` is not associated with any `then` block. This means that if the promise is rejected, the error will be caught and handled, but if the promise is resolved, nothing will happen. It would be better to have a `then` block before the `catch` block to handle the resolution of the promise.